packages/uiweb/src/lib/components/chat/ChatProfile/AddWalletContent.tsx

```diff
@@ -185,12 +185,12 @@ export const AddWalletContent = ({
       <Section flex="1" alignSelf="center">
         <ModalConfirmButton
           onClick={() => {
-            // console.log(groupMembers);
             onSubmit()
           }}
           isLoading={isLoading}
           memberListCount={memberList?.length > 0}
           theme={theme}
+          disabled={isLoading || groupMembers.length === 0}
```

Comment:
why is this needed?

Reply:
for making the create group button disable when the group is being created and to add a condition that there should be at least one more wallet address added here